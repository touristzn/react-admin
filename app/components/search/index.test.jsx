import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Search from './index'

let wrapper

describe('<Search />', () => {
  const props = {
    placeholder: '请输入内容',
    onChange: jest.fn(),
  }

  beforeEach(() => {
    wrapper = shallow(
      <Search icon {...props} />
    )
  })

  it('renders correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})