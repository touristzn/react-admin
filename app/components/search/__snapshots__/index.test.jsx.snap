// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> renders correctly 1`] = `
<div
  className="input"
>
  <input
    className="hasIcon"
    onChange={[Function]}
    placeholder="请输入内容"
    type="text"
    value=""
  />
</div>
`;
